tests/local_providers/allocine_stocks_test.py

```diff
@@ -272,9 +272,156 @@ def test_should_create_one_product_and_one_offer_with_movie_info(self, mock_call
+                                    "backlink": {
+                                        "url": r"http:\/\/www.allocine.fr\/film\/fichefilm-255951\/casting\/",
+                                        "label": "Casting complet du film sur AlloCin\u00e9"
+                                    },
+                                    "edges": []
+                                },
+                                "countries": [
+                                    {
+                                        "name": "Iran",
+                                        "alpha3": "IRN"
+                                    }
+                                ],
+                                "genres": [
+                                    "ANIMATION",
+                                    "FAMILY"
+                                ],
+                                "companies": []
+                            },
+                            "showtimes": [
+                                {
+                                    "startsAt": "2019-10-29T10:30:00",
+                                    "diffusionVersion": "ORIGINAL"
+                                },
+                                {
+                                    "startsAt": "2019-10-29T10:30:00",
+                                    "diffusionVersion": "DUBBED"
+                                },
+                                {
+                                    "startsAt": "2019-10-29T14:30:00",
+                                    "diffusionVersion": "ORIGINAL"
+                                },
+                                {
+                                    "startsAt": "2019-10-29T14:30:00",
+                                    "diffusionVersion": "DUBBED"
+                                }
+                            ]
+                        }
+                    }])
+
+                offerer = create_offerer(siren='775671464')
+                venue = create_venue(offerer, name='Cinema Allocine', siret='77567146400110',
+                                     booking_email='toto@toto.com')
+                PcObject.save(venue)
+
+                allocine_provider = get_provider_by_local_class('AllocineStocks')
+                allocine_provider.isActive = True
+                venue_provider = create_venue_provider(venue, allocine_provider,
+                                                       venue_id_at_offer_provider=theater_token)
+                PcObject.save(venue_provider)
+
+                allocine_stocks_provider = AllocineStocks(venue_provider)
+
+                # When
+                allocine_stocks_provider.updateObjects()
+
+                # Then
+                created_offers = Offer.query.all()
+                created_product = Product.query.one()
+
+                assert len(created_offers) == 2
+
+                original_version_offer = created_offers[0]
+                assert original_version_offer.bookingEmail == 'toto@toto.com'
```

Comment:
Je sais que c'est l'habitude en Python (et encore, j'ai des contre exemple) mais en JS, on ferait plutôt :
```suggestion
                assert original_version_offer == {...}
```
Je trouve qu'il y a moins d'écriture et donc plus lisible.
Qu'en penses tu ?

Reply:
Bon j'ai galéré pendant 30 min. J'ai des tests qui fail parce que certains `dict` ne possèdent pas d' `id`. J'ai laissé tomber pour éviter de perdre du temps